<!DOCTYPE html>
<html lang="en">

  <head>
    
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>

  Jay


</title>
<meta name="description" content="Computational structural biology postdoc @ Biozentrum, University of Basel
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;0,700;1,400&family=Playfair+Display:wght@500&display=swap" rel="stylesheet">
<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css" />

<!-- Styles -->

<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>J</text></svg>">

<link rel="stylesheet" href="/assets/css/main.css">
<link rel="canonical" href="https://ninjani.github.io/">



  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
        <!-- Social Icons -->
        <div class="navbar-brand social">
          <a href="mailto:%6A%61%6E%61%6E%69.%64%75%72%61%69%72%61%6A@%75%6E%69%62%61%73.%63%68" title="email"><i class="fas fa-envelope"></i></a>
<a href="https://orcid.org/0000-0002-1698-4556" title="ORCID" target="_blank" rel="noopener noreferrer"><i class="ai ai-orcid"></i></a>
<a href="https://scholar.google.com/citations?user=PQ-my-kAAAAJ&amp;hl" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a>


<a href="https://github.com/Ninjani" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a>

<a href="https://twitter.com/janani_hex" title="Twitter" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a>


<a href="https://www.biozentrum.unibas.ch/research/research-groups/project-leaders-a-z/overview/unit/research-group-durairaj-j" title="Work" target="_blank" rel="noopener noreferrer"><i class="fas fa-briefcase"></i></a>



<a href="https://gitlab.com/ninjani23" title="GitLab" target="_blank" rel="noopener noreferrer"><i class="fab fa-gitlab"></i></a>
<a href="https://out-of-cheese-error.netlify.app/" title="Blog" target="_blank" rel="noopener noreferrer"><i class="fas fa-cheese"></i></a>



<a href="https://Ninjani.github.io/feed.xml" title="RSS Feed"><i class="fas fa-rss-square"></i></a>

        </div>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item active">
            <a class="nav-link" href="/">
              about
              
                <span class="sr-only">(current)</span>
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/publications/">
                publications
                
              </a>
          </li>
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/teaching/">
                teaching
                
              </a>
          </li>
          
          
          
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h1 class="post-title">
     Jay
    </h1>
     <p class="desc"></p>
<h3 class="name">Janani Durairaj</h3>
  </header>

  <article>
    
    <div class="profile float-right">
      
        <picture>
    
    <source media="(max-width: 480px)" srcset="/assets/resized/prof_pic-480x480.jpg">
    
    <source media="(max-width: 800px)" srcset="/assets/resized/prof_pic-800x800.jpg">
    
    <img class="img-fluid z-depth-1 rounded" src="/assets/img/prof_pic.jpg" alt="prof_pic.jpg">
</source></source></picture>

      
      
    </div>
    

    <div class="clearfix justify">
      <p>I’m a project leader working on <a href="https://www.biozentrum.unibas.ch/research/research-groups/project-leaders-a-z/overview/unit/research-group-durairaj-j" target="_blank" rel="noopener noreferrer">bringing biochemical context to protein deep learning models</a>, in Torsten Schwede’s <em><a href="https://www.biozentrum.unibas.ch/research/research-groups/research-groups-a-z/overview/unit/research-group-torsten-schwede" target="_blank" rel="noopener noreferrer">Protein Bioinformatics in 3D</a></em> research group at the Biozentrum, University of Basel.
I work at the interface of protein structural bioinformatics and machine learning, developing efficient algorithms and 
software to represent, explore and understand protein structures for applications such as enzyme engineering, drug discovery and biomedicine.</p>

<p>The Picky Binders team:
<br>
<img src="assets/img/picky_binders.png" width="500"></p>

<p>Find me on Bluesky as <a href="https://bsky.app/profile/ninjani.bsky.social" target="_blank" rel="noopener noreferrer">@ninjani.bsky.social</a></p>

    </div>

    
      <div class="news">
  <h2>news</h2>
  
    <div class="table-responsive">
      <table class="table table-sm">
      
      
        <tr>
          <th scope="row">Nov 1, 2024</th>
          <td>
            
              Started as an <a href="https://www.biozentrum.unibas.ch/news/detail/two-new-snsf-ambizione-fellows-at-the-biozentrum" target="_blank" rel="noopener noreferrer">Ambizione Project Leader</a>, bringing <a href="https://www.biozentrum.unibas.ch/research/research-groups/project-leaders-a-z/overview/unit/research-group-durairaj-j" target="_blank" rel="noopener noreferrer">biochemical context to protein deep learning models</a>

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Aug 30, 2024</th>
          <td>
            
              Spoke about <a href="https://plinder.sh" target="_blank" rel="noopener noreferrer">PLINDER</a> at the <a href="https://www.youtube.com/watch?v=DLniEFBhkWA" target="_blank" rel="noopener noreferrer">Generative AI in Drug Discovery series</a>

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Apr 9, 2024</th>
          <td>
            
              The Protein Universe Atlas was selected as one of the <a href="https://www.sib.swiss/news/discover-the-sib-remarkable-outputs-2023" target="_blank" rel="noopener noreferrer">SIB Remarkable Outputs 2023</a>

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Sep 13, 2023</th>
          <td>
            
              <a href="https://www.nature.com/articles/d41586-023-02892-z" target="_blank" rel="noopener noreferrer">‘A Pandora’s box’: map of protein-structure families delights scientists</a>

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Jun 28, 2022</th>
          <td>
            
              Winner of the <a href="https://www.aanmelder.nl/biosb2022/yia" target="_blank" rel="noopener noreferrer">BioSB Young Investigator Award</a> 
“for a young researcher who has significantly contributed to the bioinformatics or the systems biology research community in the Netherlands.”

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Nov 6, 2021</th>
          <td>
            
              Kickstarted a structural bioinformatics newsletter with the group: <a href="https://folded-weekly.netlify.app/" target="_blank" rel="noopener noreferrer">FoldedWeekly</a>

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Sep 15, 2021</th>
          <td>
            
              Defended my PhD thesis entitled “Computational approaches to discover novel enzymes for fragrance and flavour”, <a href="https://en.wikipedia.org/wiki/Latin_honors#Netherlands" target="_blank" rel="noopener noreferrer">cum laude</a>!

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Sep 1, 2021</th>
          <td>
            
              Started as a postdoc at the <a href="https://www.biozentrum.unibas.ch/research/research-groups/research-groups-a-z/overview/unit/research-group-torsten-schwede" target="_blank" rel="noopener noreferrer">Protein Bioinformatics in 3D research group</a>

            
          </td>
        </tr>
      
        <tr>
          <th scope="row">Jun 30, 2021</th>
          <td>
            
              <a href="https://anchor.fm/chelonia/episodes/Interview-series-of-LIGATE-project-with-Janani-Durairaj-e14vl4h" target="_blank" rel="noopener noreferrer">Talked about</a> what I’ll be working on as part of the <a href="https://www.ligateproject.eu/" target="_blank" rel="noopener noreferrer">LIGATE project</a>

            
          </td>
        </tr>
      
      </table>
    </div>
  
</div>

    

    
      <div class="publications">
  <h2>selected publications</h2>
  <h2 class="bibliography">working papers</h2>
<h3 class="bibliography">2025</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="vskrinjar2025have" class="col-sm-8">
    
      <div class="title">Have protein-ligand co-folding methods moved beyond memorisation?</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Peter  Škrinjar,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jérôme  Eberhardt,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Torsten  Schwede
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1101/2025.02.03.636309" role="button" target="_blank" rel="noopener noreferrer">Preprint</a>
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/plinder-org/runs-n-poses" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Deep learning has driven major breakthroughs in protein structure prediction, however the next critical advance is accurately predicting how proteins interact with other molecules, especially small molecule ligands, to enable real-world applications such as drug discovery and design. Recent deep learning all-atom methods have been built to address this challenge, but evaluating their performance on the prediction of protein-ligand complexes has been inconclusive due to the lack of relevant benchmarking datasets. Here we present a comprehensive evaluation of four leading all-atom cofolding deep learning methods using our newly introduced benchmark dataset Runs N’ Poses, which comprises 2,600 high-resolution protein-ligand systems released after the training cutoff used by these methods. We demonstrate that current co-folding approaches largely memorise ligand poses from their training data, hindering their use for de novo drug design. This limitation is especially pronounced for ligands that have only been seen binding in one pocket, whereas more promiscuous ligands such as cofactors show moderately improved performance. With this work and benchmark dataset, we aim to accelerate progress in the field by allowing for a more realistic assessment of the current state-of-the-art deep learning methods for predicting protein-ligand interactions.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
  </div>
</div></li></ol>
<h2 class="bibliography">journal articles</h2>
<h3 class="bibliography">2023</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Nature</abbr>
    
  
  </div>

  <div id="durairaj2023uncovering" class="col-sm-8">
    
      <div class="title">Uncovering new families and folds in the natural protein universe</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andrew M  Waterhouse,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Toomas  Mets,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tetiana  Brodiazhenko,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Minhal  Abdullah,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerardo  Tauriello,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Antonina  Andreeva,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Alex  Bateman,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tanel  Tensor,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Vasili  Hauryliuk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Joana  Pereira
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Nature
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/ProteinUniverseAtlas/AFDB90v4" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://uniprot3d.org/atlas/AFDB90v4" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>We are now entering a new era in protein sequence and structure annotation, with hundreds of millions of predicted protein structures made available through the AlphaFold database. These models cover nearly all proteins that are known, including those challenging to annotate for function or putative biological role using standard homology-based approaches. In this study, we examine the extent to which the AlphaFold database has structurally illuminated this "dark matter" of the natural protein universe at high predicted accuracy. We further describe the protein diversity that these models cover as an annotated interactive sequence similarity network. By searching for novelties from sequence, structure, and semantic perspectives, we uncovered the beta-flower fold, added multiple protein families to Pfam database, and experimentally demonstrate that one of these belongs to a new superfamily of translation-targeting toxin-antitoxin systems, TumE-TumA. This work underscores the value of large-scale efforts in identifying, annotating, and prioritising novel protein families. By leveraging the recent deep learning revolution in protein bioinformatics, we can now shed light into uncharted areas of the protein universe at an unprecedented scale, paving the way to innovations in life sciences and biotechnology.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">durairaj2023uncovering</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Uncovering new families and folds in the natural protein universe}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj, Janani and Waterhouse, Andrew M and Mets, Toomas and Brodiazhenko, Tetiana and Abdullah, Minhal and Studer, Gabriel and Tauriello, Gerardo and Akdel, Mehmet and Andreeva, Antonina and Bateman, Alex and Tensor, Tanel and Hauryliuk, Vasili and Schwede, Torsten and Pereira, Joana}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--3}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-023-06622-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li></ol>
<h2 class="bibliography">conference articles &amp; others</h2>
<h3 class="bibliography">2024</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">ICML</abbr>
    
  
  </div>

  <div id="durairaj24plinder" class="col-sm-8">
    
      <div class="title">PLINDER: The Protein-Ligand Interactions Dataset and Evaluation Resource</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Yusuf  Adeshina<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Zhonglin  Cao,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Xuejin  Zhang,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Vladas  Oleinikovas,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Thomas  Duignan,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Zachary  McClure,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Xavier  Robin,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Daniel  Kovtun,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Emanuele  Rossi,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Guoqing  Zhou,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Srimukh  Veccham,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Clemens  Isert,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Yuxing  Peng,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Prabindh  Sundareson,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriele  Corso,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Hannes  Stärk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerardo  Tauriello,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Zachary  Carpenter,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Michael  Bronstein,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Emine  Kucukbenli,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Luca  Naef
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In ICML’24 Workshop ML for Life and Material Science: From Theory to Industry Applications
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1101/2024.07.17.603955" role="button" target="_blank" rel="noopener noreferrer">Preprint</a>
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/plinder-org/plinder" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://plinder.sh" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Protein-ligand interactions (PLI) are foundational to small molecule drug design. With computational methods striving towards experimental accuracy, there is a critical demand for a well-curated and diverse PLI dataset. Existing datasets are often limited in size and diversity, and commonly used evaluation sets suffer from training information leakage, hindering the realistic assessment of method generalization capabilities. To address these shortcomings, we present PLINDER, the largest and most annotated dataset to date, comprising 449,383 PLI systems, each with over 500 annotations, similarity metrics at protein, pocket, interaction and ligand levels, and paired unbound (apo) and predicted structures. We propose an approach to generate training and evaluation splits that minimizes task-specific leakage and maximizes test set quality, and compare the resulting performance of DiffDock when retrained with different kinds of splits.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
  </div>
</div></li></ol>
</div>

    

    
  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    © Copyright 2025 Janani  Durairaj.
    Powered by <a href="http://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>.

    
    
    Last updated: February 14, 2025.
    
  </div>
</footer>



  </body>

  <!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Medium Zoom JS -->
<script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
<script src="/assets/js/zoom.js"></script>


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>

  
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  





</html>
