<!DOCTYPE html>
<html lang="en">

  <head>
    
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>

  Jay


  | publications

</title>
<meta name="description" content="Computational structural biology postdoc @ Biozentrum, University of Basel
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;0,700;1,400&family=Playfair+Display:wght@500&display=swap" rel="stylesheet">
<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css" />

<!-- Styles -->

<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>J</text></svg>">

<link rel="stylesheet" href="/assets/css/main.css">
<link rel="canonical" href="https://ninjani.github.io/publications/">



  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="https://Ninjani.github.io/">
       Jay
      </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">
              about
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item active">
              <a class="nav-link" href="/publications/">
                publications
                
                <span class="sr-only">(current)</span>
                
              </a>
          </li>
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/teaching/">
                teaching
                
              </a>
          </li>
          
          
          
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h1 class="post-title">publications</h1>
    <p class="post-description"><nobr><em>*</em></nobr> denotes joint lead authorship, <nobr><em>#</em></nobr> denotes joint corresponding authorship</p>
  </header>

  <article>
    <!-- _pages/publications.md -->
<div class="publications">

<h2 class="bibliography">working papers</h2>
<h3 class="bibliography">2025</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="vskrinjar2025have" class="col-sm-8">
    
      <div class="title">Have protein-ligand co-folding methods moved beyond memorisation?</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Peter  Škrinjar,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jérôme  Eberhardt,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Torsten  Schwede
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1101/2025.02.03.636309" role="button" target="_blank" rel="noopener noreferrer">Preprint</a>
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/plinder-org/runs-n-poses" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Deep learning has driven major breakthroughs in protein structure prediction, however the next critical advance is accurately predicting how proteins interact with other molecules, especially small molecule ligands, to enable real-world applications such as drug discovery and design. Recent deep learning all-atom methods have been built to address this challenge, but evaluating their performance on the prediction of protein-ligand complexes has been inconclusive due to the lack of relevant benchmarking datasets. Here we present a comprehensive evaluation of four leading all-atom cofolding deep learning methods using our newly introduced benchmark dataset Runs N’ Poses, which comprises 2,600 high-resolution protein-ligand systems released after the training cutoff used by these methods. We demonstrate that current co-folding approaches largely memorise ligand poses from their training data, hindering their use for de novo drug design. This limitation is especially pronounced for ligands that have only been seen binding in one pocket, whereas more promiscuous ligands such as cofactors show moderately improved performance. With this work and benchmark dataset, we aim to accelerate progress in the field by allowing for a more realistic assessment of the current state-of-the-art deep learning methods for predicting protein-ligand interactions.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
  </div>
</div></li></ol>
<h2 class="bibliography">journal articles</h2>
<h3 class="bibliography">2024</h3>
<ol class="bibliography">
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Bioinformatics</abbr>
    
  
  </div>

  <div id="pantolini2024embedding" class="col-sm-8">
    
      <div class="title">Embedding-based alignment: combining protein language models with dynamic programming alignment to detect structural similarities in the twilight-zone</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Lorenzo  Pantolini,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Joana  Pereira,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerardo  Tauriello,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Torsten  Schwede
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Bioinformatics
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://academic.oup.com/bioinformatics/article/40/1/btad786" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Motivation: Language models are routinely used for text classification and generative tasks. Recently, the same architectures were applied to protein sequences, unlocking powerful new approaches in the bioinformatics field. Protein language models (pLMs) generate high-dimensional embeddings on a per-residue level and encode a “semantic meaning” of each individual amino acid in the context of the full protein sequence. These representations have been used as a starting point for downstream learning tasks and, more recently, for identifying distant homologous relationships between proteins. Results: In this work, we introduce a new method that generates embedding-based protein sequence alignments (EBA) and show how these capture structural similarities even in the twilight zone, outperforming both classical methods as well as other approaches based on pLMs. The method shows excellent accuracy despite the absence of training and parameter optimization. We demonstrate that the combination of pLMs with alignment methods is a valuable approach for the detection of relationships between proteins in the twilight-zone. Availability and implementation: The code to run EBA and reproduce the analysis described in this article is available at: https://git.scicore.unibas.ch/schwede/EBA and https://git.scicore.unibas.ch/schwede/eba_benchmark.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pantolini2024embedding</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Embedding-based alignment: combining protein language models with dynamic programming alignment to detect structural similarities in the twilight-zone}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pantolini, Lorenzo and Studer, Gabriel and Pereira, Joana and Durairaj, Janani and Tauriello, Gerardo and Schwede, Torsten}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Bioinformatics}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{btad786}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bioinformatics/btad786}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Msphere</abbr>
    
  
  </div>

  <div id="durairaj2024structural" class="col-sm-8">
    
      <div class="title">Structural implications of BK polyomavirus sequence variations in the major viral capsid protein Vp1 and large T-antigen: a computational study</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Océane M  Follonier,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Karoline  Leuzinger,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Leila T  Alexander,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Maud  Wilhelm,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Joana  Pereira,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Caroline A  Hillenbrand,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Fabian H  Weissbach,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Hans H  Hirsch
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Msphere
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1128/msphere.00799-23" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>BK polyomavirus (BKPyV) is a double-stranded DNA virus causing nephropathy, hemorrhagic cystitis, and urothelial cancer in transplant patients. The BKPyV-encoded capsid protein Vp1 and large T-antigen (LTag) are key targets of neutralizing antibodies and cytotoxic T-cells, respectively. Our single-center data suggested that variability in Vp1 and LTag may contribute to failing BKPyV-specific immune control, and impact vaccine design. We therefore analyzed all available entries in GenBank (1516 VP1; 742 LTAG) and explored potential structural effects using computational approaches. BKPyV-genotype (gt)1 was found in 71.18% of entries, followed by BKPyV-gt4 (19.26%), BKPyV-gt2 (8.11%) and BKPyV-gt3 (1.45%), but rates differed according to country and specimen type. Vp1-mutations matched a serotype different than the assigned one or were serotype-independent in 43%, 18% affected more than one amino acid. Notable Vp1-mutations altered antibody-binding domains, interactions with sialic acid receptors, or were predicted to change conformation. LTag-sequences were more conserved, with only 16 mutations detectable in more than one entry and without significant effects on LTag-structure or interaction domains. However, LTag changes were predicted to affect HLA-class I presentation of immunodominant 9mers to cytotoxic T-cells. These global data strengthen single center observations and specifically our earlier findings revealing mutant 9mer epitopes conferring immune escape from HLA-I cytotoxic T cells. We conclude that variability of BKPyV-Vp1 and LTag may have important implications for diagnostic assays assessing BKPyV-specific immune control and for vaccine design.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">durairaj2024structural</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Structural implications of BK polyomavirus sequence variations in the major viral capsid protein Vp1 and large T-antigen: a computational study}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj, Janani and Follonier, Oc{\'e}ane M and Leuzinger, Karoline and Alexander, Leila T and Wilhelm, Maud and Pereira, Joana and Hillenbrand, Caroline A and Weissbach, Fabian H and Schwede, Torsten and Hirsch, Hans H}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Msphere}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e00799--23}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Am Soc Microbiol}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1128/msphere.00799-23}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
</ol>
<h3 class="bibliography">2023</h3>
<ol class="bibliography">
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Nature</abbr>
    
  
  </div>

  <div id="durairaj2023uncovering" class="col-sm-8">
    
      <div class="title">Uncovering new families and folds in the natural protein universe</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andrew M  Waterhouse,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Toomas  Mets,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tetiana  Brodiazhenko,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Minhal  Abdullah,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerardo  Tauriello,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Antonina  Andreeva,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Alex  Bateman,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tanel  Tensor,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Vasili  Hauryliuk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Joana  Pereira
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Nature
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/ProteinUniverseAtlas/AFDB90v4" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://uniprot3d.org/atlas/AFDB90v4" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>We are now entering a new era in protein sequence and structure annotation, with hundreds of millions of predicted protein structures made available through the AlphaFold database. These models cover nearly all proteins that are known, including those challenging to annotate for function or putative biological role using standard homology-based approaches. In this study, we examine the extent to which the AlphaFold database has structurally illuminated this "dark matter" of the natural protein universe at high predicted accuracy. We further describe the protein diversity that these models cover as an annotated interactive sequence similarity network. By searching for novelties from sequence, structure, and semantic perspectives, we uncovered the beta-flower fold, added multiple protein families to Pfam database, and experimentally demonstrate that one of these belongs to a new superfamily of translation-targeting toxin-antitoxin systems, TumE-TumA. This work underscores the value of large-scale efforts in identifying, annotating, and prioritising novel protein families. By leveraging the recent deep learning revolution in protein bioinformatics, we can now shed light into uncharted areas of the protein universe at an unprecedented scale, paving the way to innovations in life sciences and biotechnology.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">durairaj2023uncovering</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Uncovering new families and folds in the natural protein universe}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj, Janani and Waterhouse, Andrew M and Mets, Toomas and Brodiazhenko, Tetiana and Abdullah, Minhal and Studer, Gabriel and Tauriello, Gerardo and Akdel, Mehmet and Andreeva, Antonina and Bateman, Alex and Tensor, Tanel and Hauryliuk, Vasili and Schwede, Torsten and Pereira, Joana}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--3}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-023-06622-3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">NRDD</abbr>
    
  
  </div>

  <div id="mullowneyArtificialIntelligenceNatural2023" class="col-sm-8">
    
      <div class="title">Artificial Intelligence for Natural Product Drug Discovery</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Michael W.  Mullowney,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Katherine R.  Duncan,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Somayah S.  Elsayed,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Neha  Garg,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Justin J. J.  van der Hooft,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Nathaniel I.  Martin,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  David  Meijer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Barbara R.  Terlouw,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Friederike  Biermann,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Kai  Blin,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Marina  Gorostiola González,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Eric J. N.  Helfrich,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Florian  Huber,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Stefan  Leopold-Messer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Kohulan  Rajan,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tristan  de Rond,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jeffrey A.  van Santen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Maria  Sorokina,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Marcy J.  Balunas,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mehdi A.  Beniddir,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Doris A.  van Bergeijk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Laura M.  Carroll,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Chase M.  Clark,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Djork-Arné  Clevert,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Chris A.  Dejong,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Chao  Du,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Scarlet  Ferrinho,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Francesca  Grisoni,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Albert  Hofstetter,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Willem  Jespers,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Olga V.  Kalinina,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Satria A.  Kautsar,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Hyunwoo  Kim,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tiago F.  Leao,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Joleen  Masschelein,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Evan R.  Rees,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Raphael  Reher,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Daniel  Reker,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Philippe  Schwaller,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Marwin  Segler,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Michael A.  Skinnider,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Allison S.  Walker,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Egon L.  Willighagen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Barbara  Zdrazil,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Nadine  Ziemert,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Rebecca J. M.  Goss,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Pierre  Guyomard,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andrea  Volkamer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  William H.  Gerwick,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Hyun Uk  Kim,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Rolf  Müller,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gilles P.  van Wezel,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerard J. P.  van Westen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Anna K. H.  Hirsch,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Roger G.  Linington,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Serina L.  Robinson,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Marnix H.  Medema
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Nature Reviews Drug Discovery
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Developments in computational omics technologies have provided new means to access the hidden diversity of natural products, unearthing new potential for drug discovery. In parallel, artificial intelligence approaches such as machine learning have led to exciting developments in the computational drug design field, facilitating biological activity prediction and de novo drug design for molecular targets of interest. Here, we describe current and future synergies between these developments to effectively identify drug candidates from the plethora of molecules produced by nature. We also discuss how to address key challenges in realizing the potential of these synergies, such as the need for high-quality datasets to train deep learning algorithms and appropriate strategies for algorithm validation.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mullowneyArtificialIntelligenceNatural2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Artificial Intelligence for Natural Product Drug Discovery}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mullowney, Michael W. and Duncan, Katherine R. and Elsayed, Somayah S. and Garg, Neha and {van der Hooft}, Justin J. J. and Martin, Nathaniel I. and Meijer, David and Terlouw, Barbara R. and Biermann, Friederike and Blin, Kai and Durairaj, Janani and Gorostiola Gonz{\'a}lez, Marina and Helfrich, Eric J. N. and Huber, Florian and {Leopold-Messer}, Stefan and Rajan, Kohulan and {de Rond}, Tristan and {van Santen}, Jeffrey A. and Sorokina, Maria and Balunas, Marcy J. and Beniddir, Mehdi A. and {van Bergeijk}, Doris A. and Carroll, Laura M. and Clark, Chase M. and Clevert, Djork-Arn{\'e} and Dejong, Chris A. and Du, Chao and Ferrinho, Scarlet and Grisoni, Francesca and Hofstetter, Albert and Jespers, Willem and Kalinina, Olga V. and Kautsar, Satria A. and Kim, Hyunwoo and Leao, Tiago F. and Masschelein, Joleen and Rees, Evan R. and Reher, Raphael and Reker, Daniel and Schwaller, Philippe and Segler, Marwin and Skinnider, Michael A. and Walker, Allison S. and Willighagen, Egon L. and Zdrazil, Barbara and Ziemert, Nadine and Goss, Rebecca J. M. and Guyomard, Pierre and Volkamer, Andrea and Gerwick, William H. and Kim, Hyun Uk and M{\"u}ller, Rolf and {van Wezel}, Gilles P. and {van Westen}, Gerard J. P. and Hirsch, Anna K. H. and Linington, Roger G. and Robinson, Serina L. and Medema, Marnix H.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Reviews Drug Discovery}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1474-1784}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41573-023-00774-7}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Proteins</abbr>
    
  
  </div>

  <div id="alexanderProteinTargetHighlights" class="col-sm-8">
    
      <div class="title">Protein target highlights in CASP15: Analysis of models by structure providers</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Leila T.  Alexander<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andriy  Kryshtafovych,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Luciano A.  Abriata,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Yusupha  Bayo,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gira  Bhabha,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Cécile  Breyton,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Simon G.  Caulton,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  James  Chen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Séraphine  Degroux,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Damian C.  Ekiert,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Benedikte S.  Erlandsen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Peter L.  Freddolino,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dominic  Gilzer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Chris  Greening,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jonathan M.  Grimes,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Rhys  Grinter,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Manickam  Gurusaran,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Marcus D.  Hartmann,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Charlie J.  Hitchman,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jeremy R.  Keown,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Ashleigh  Kropp,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Petri  Kursula,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andrew L.  Lovering,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Bruno  Lemaitre,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andrea  Lia,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Shiheng  Liu,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Maria  Logotheti,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Shuze  Lu,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Sigurbjörn  Markússon,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mitchell D.  Miller,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  George  Minasov,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Hartmut H.  Niemann,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Felipe  Opazo,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  George N.  Phillips Jr,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Owen R.  Davies,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Samuel  Rommelaere,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Monica  Rosas-Lemus,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Pietro  Roversi,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Karla  Satchell,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Nathan  Smith,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mark A.  Wilson,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Kuan-Lin  Wu,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Xian  Xia,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Han  Xiao,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Wenhua  Zhang,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Z. Hong  Zhou,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Krzysztof  Fidelis,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Maya  Topf,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  John  Moult,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Torsten  Schwede
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Proteins: Structure, Function, and Bioinformatics
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.26545" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>We present an in-depth analysis of selected CASP15 targets, focusing on their biological and functional significance. The authors of the structures identify and discuss key protein features and evaluate how effectively these aspects were captured in the submitted predictions. While the overall ability to predict three-dimensional protein structures continues to impress, reproducing uncommon features not previously observed in experimental structures is still a challenge. Furthermore, instances with conformational flexibility and large multimeric complexes highlight the need for novel scoring strategies to better emphasize biologically relevant structural regions. Looking ahead, closer integration of computational and experimental techniques will play a key role in determining the next challenges to be unraveled in the field of structural molecular biology.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">alexanderProteinTargetHighlights</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Protein target highlights in {CASP15}: {Analysis} of models by structure providers}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{© 2023 The Authors. Proteins: Structure, Function, and Bioinformatics published by Wiley Periodicals LLC.}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1097-0134}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Protein target highlights in {CASP15}}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/prot.26545}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proteins: Structure, Function, and Bioinformatics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Alexander&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Leila T. and Durairaj&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Janani and Kryshtafovych, Andriy and Abriata, Luciano A. and Bayo, Yusupha and Bhabha, Gira and Breyton, Cécile and Caulton, Simon G. and Chen, James and Degroux, Séraphine and Ekiert, Damian C. and Erlandsen, Benedikte S. and Freddolino, Peter L. and Gilzer, Dominic and Greening, Chris and Grimes, Jonathan M. and Grinter, Rhys and Gurusaran, Manickam and Hartmann, Marcus D. and Hitchman, Charlie J. and Keown, Jeremy R. and Kropp, Ashleigh and Kursula, Petri and Lovering, Andrew L. and Lemaitre, Bruno and Lia, Andrea and Liu, Shiheng and Logotheti, Maria and Lu, Shuze and Markússon, Sigurbjörn and Miller, Mitchell D. and Minasov, George and Niemann, Hartmut H. and Opazo, Felipe and Phillips Jr, George N. and Davies, Owen R. and Rommelaere, Samuel and Rosas-Lemus, Monica and Roversi, Pietro and Satchell, Karla and Smith, Nathan and Wilson, Mark A. and Wu, Kuan-Lin and Xia, Xian and Xiao, Han and Zhang, Wenhua and Zhou, Z. Hong and Fidelis, Krzysztof and Topf, Maya and Moult, John and Schwede, Torsten}</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Proteins</abbr>
    
  
  </div>

  <div id="leemannAutomatedBenchmarkingCombined" class="col-sm-8">
    
      <div class="title">Automated benchmarking of combined protein structure and ligand conformation prediction</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Michèle  Leemann,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Ander  Sagasta,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jerome  Eberhardt,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Xavier  Robin,
                
              
            
          
        
          
          
          
          

          
          
            
              
                and <em class="author">Janani  Durairaj</em>
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Proteins: Structure, Function, and Bioinformatics
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The prediction of protein-ligand complexes (PLC), using both experimental and predicted structures, is an active and important area of research, underscored by the inclusion of the Protein-Ligand Interaction category in the latest round of the Critical Assessment of Protein Structure Prediction experiment CASP15. The prediction task in CASP15 consisted of predicting both the three-dimensional structure of the receptor protein as well as the position and conformation of the ligand. This paper addresses the challenges and proposed solutions for devising automated benchmarking techniques for PLC prediction. The reliability of experimentally solved PLC as ground truth reference structures is assessed using various validation criteria. Similarity of PLC to previously released complexes are employed to judge PLC diversity and the difficulty of a PLC as a prediction target. We show that the commonly used PDBBind time-split test-set is inappropriate for comprehensive PLC evaluation, with state-of-the-art tools showing conflicting results on a more representative and high quality dataset constructed for benchmarking purposes. We also show that redocking on crystal structures is a much simpler task than docking into predicted protein models, demonstrated by the two PLC-prediction-specific scoring metrics created. Finally, we introduce a fully automated pipeline that predicts PLC and evaluates the accuracy of the protein structure, ligand pose, and protein–ligand interactions.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">leemannAutomatedBenchmarkingCombined</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Automated benchmarking of combined protein structure and ligand conformation prediction}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{© 2023 The Authors. Proteins: Structure, Function, and Bioinformatics published by Wiley Periodicals LLC.}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1097-0134}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.26605}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/prot.26605}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-27}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proteins: Structure, Function, and Bioinformatics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Leemann, Michèle and Sagasta, Ander and Eberhardt, Jerome and Schwede, Torsten and Robin, Xavier and Durairaj, Janani}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{\_eprint: https://onlinelibrary.wiley.com/doi/  keywords = {3D structure prediction, CASP15, protein structure, protein-ligand complexes},
  year = {2023},
}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Proteins</abbr>
    
  
  </div>

  <div id="robinAssessmentProteinLigand" class="col-sm-8">
    
      <div class="title">Assessment of protein–ligand complexes in CASP15</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Xavier  Robin,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jerome  Eberhardt,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and W. Patrick  Walters
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Proteins: Structure, Function, and Bioinformatics
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.26601" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>CASP15 introduced a new category, ligand prediction, where participants were provided with a protein or nucleic acid sequence, SMILES line notation, and stoichiometry for ligands and tasked with generating computational models for the three-dimensional structure of the corresponding protein–ligand complex. These models were subsequently compared with experimental structures determined by x-ray crystallography or cryoEM. To assess these predictions, two novel scores were developed. The Binding-Site Superposed, Symmetry-Corrected Pose Root Mean Square Deviation (BiSyRMSD) evaluated the absolute deviations of the models from the experimental structures. At the same time, the Local Distance Difference Test for Protein–Ligand Interactions (lDDT-PLI) assessed the ability of models to reproduce the protein–ligand interactions in the experimental structures. The ligands evaluated in this challenge range from single-atom ions to large flexible organic molecules. More than 1800 submissions were evaluated for their ability to predict 23 different protein–ligand complexes. Overall, the best models could faithfully reproduce the geometries of more than half of the prediction targets. The ligands’ size and flexibility were the primary factors influencing the predictions’ quality. Small ions and organic molecules with limited flexibility were predicted with high fidelity, while reproducing the binding poses of larger, flexible ligands proved more challenging.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">robinAssessmentProteinLigand</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Assessment of protein–ligand complexes in {CASP15}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{© 2023 Wiley Periodicals LLC.}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1097-0134}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/prot.26601}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proteins: Structure, Function, and Bioinformatics}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Robin, Xavier and Studer, Gabriel and Durairaj, Janani and Eberhardt, Jerome and Schwede, Torsten and Walters, W. Patrick}</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Proteins</abbr>
    
  
  </div>

  <div id="kryshtafovychNewPredictionCategories" class="col-sm-8">
    
      <div class="title">New prediction categories in CASP15</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Andriy  Kryshtafovych,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Maciej  Antczak,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Marta  Szachniuk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tomasz  Zok,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Rachael C.  Kretsch,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Ramya  Rangan,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Phillip  Pham,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Rhiju  Das,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Xavier  Robin,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jerome  Eberhardt,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Aaron  Sweeney,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Maya  Topf,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Krzysztof  Fidelis,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and John  Moult
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Proteins: Structure, Function, and Bioinformatics
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.26515" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Prediction categories in the Critical Assessment of Structure Prediction (CASP) experiments change with the need to address specific problems in structure modeling. In CASP15, four new prediction categories were introduced: RNA structure, ligand-protein complexes, accuracy of oligomeric structures and their interfaces, and ensembles of alternative conformations. This paper lists technical specifications for these categories and describes their integration in the CASP data management system.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">kryshtafovychNewPredictionCategories</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{New prediction categories in {CASP15}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{© 2023 The Authors. Proteins: Structure, Function, and Bioinformatics published by Wiley Periodicals LLC.}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1097-0134}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/prot.26515}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{n/a}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proteins: Structure, Function, and Bioinformatics}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kryshtafovych, Andriy and Antczak, Maciej and Szachniuk, Marta and Zok, Tomasz and Kretsch, Rachael C. and Rangan, Ramya and Pham, Phillip and Das, Rhiju and Robin, Xavier and Studer, Gabriel and Durairaj, Janani and Eberhardt, Jerome and Sweeney, Aaron and Topf, Maya and Schwede, Torsten and Fidelis, Krzysztof and Moult, John}</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Science</abbr>
    
  
  </div>

  <div id="liMaizeResistanceWitchweed2023" class="col-sm-8">
    
      <div class="title">Maize resistance to witchweed through changes in strigolactone biosynthesis</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  C.  Li,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  L.  Dong,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">J.  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  J.-C.  Guan,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  M.  Yoshimura,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  P.  Quinodoz,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  R.  Horber,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  K.  Gaus,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  J.  Li,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Y. B.  Setotaw,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  J.  Qi,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  H.  De Groote,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Y.  Wang,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  B.  Thiombiano,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  K.  Floková,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  A.  Walmsley,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  T. V.  Charnikhova,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  A.  Chojnacka,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  S. Correia de Lemos,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Y.  Ding,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  D.  Skibbe,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  K.  Hermann,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  C.  Screpanti,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  A.  De Mesmaeker,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  E. A.  Schmelz,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  A.  Menkir,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  M.  Medema,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  A. D. J.  Van Dijk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  J.  Wu,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  K. E.  Koch,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and H. J.  Bouwmeester
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Science
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.science.org/doi/full/10.1126/science.abq4775" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Maize (Zea mays) is a major staple crop in Africa, where its yield and the livelihood of millions are compromised by the parasitic witchweed Striga. Germination of Striga is induced by strigolactones exuded from maize roots into the rhizosphere. In a maize germplasm collection, we identified two strigolactones, zealactol and zealactonoic acid, which stimulate less Striga germination than the major maize strigolactone, zealactone. We then showed that a single cytochrome P450, ZmCYP706C37, catalyzes a series of oxidative steps in the maize-strigolactone biosynthetic pathway. Reduction in activity of this enzyme and two others involved in the pathway, ZmMAX1b and ZmCLAMT1, can change strigolactone composition and reduce Striga germination and infection. These results offer prospects for breeding Striga-resistant maize.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liMaizeResistanceWitchweed2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Maize resistance to witchweed through changes in strigolactone biosynthesis}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{379}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/science.abq4775}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6627}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, C. and Dong, L. and Durairaj, J. and Guan, J.-C. and Yoshimura, M. and Quinodoz, P. and Horber, R. and Gaus, K. and Li, J. and Setotaw, Y. B. and Qi, J. and De Groote, H. and Wang, Y. and Thiombiano, B. and Floková, K. and Walmsley, A. and Charnikhova, T. V. and Chojnacka, A. and Correia de Lemos, S. and Ding, Y. and Skibbe, D. and Hermann, K. and Screpanti, C. and De Mesmaeker, A. and Schmelz, E. A. and Menkir, A. and Medema, M. and Van Dijk, A. D. J. and Wu, J. and Koch, K. E. and Bouwmeester, H. J.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Publisher: American Association for the Advancement of Science}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{94--99}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">CSBJ</abbr>
    
  
  </div>

  <div id="durairajSequenceStructurebasedMachine2023" class="col-sm-8">
    
      <div class="title">Beyond sequence: Structure-based machine learning</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dick de Ridder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Aalt D. J. van Dijk
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Computational and Structural Biotechnology Journal
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.sciencedirect.com/science/article/pii/S2001037022005967" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Recent breakthroughs in protein structure prediction demarcate the start of a new era in structural bioinformatics. Combined with various advances in experimental structure determination and the uninterrupted pace at which new structures are published, this promises an age in which protein structure information is as prevalent and ubiquitous as sequence. Machine learning in protein bioinformatics has been dominated by sequence-based methods, but this is now changing to make use of the deluge of rich structural information as input. Machine learning methods making use of structures are scattered across literature and cover a number of different applications and scopes; while some try to address questions and tasks within a single protein family, others aim to capture characteristics across all available proteins. In this review, we look at the variety of structure-based machine learning approaches, how structures can be used as input, and typical applications of these approaches in protein biology. We also discuss current challenges and opportunities in this all-important and increasingly popular field.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">durairajSequenceStructurebasedMachine2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Beyond sequence: {Structure}-based machine learning}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{21}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2001-0370}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Beyond sequence}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.csbj.2022.12.039}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computational and Structural Biotechnology Journal}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj, Janani and de Ridder, Dick and van Dijk, Aalt D. J.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Deep learning, Machine learning, Protein structures}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{630--643}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
</ol>
<h3 class="bibliography">2022</h3>
<ol class="bibliography">
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">NSMB</abbr>
    
  
  </div>

  <div id="akdelStructuralBiologyCommunity2022b" class="col-sm-8">
    
      <div class="title">A structural biology community assessment of AlphaFold2 applications</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Douglas E V  Pires<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Eduard  Porta Pardo<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jürgen  Jänes<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Arthur O  Zalevsky<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Bálint  Mészáros<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Patrick  Bryant<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Lydia L.  Good<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Roman A  Laskowski<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriele  Pozzati,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Aditi  Shenoy,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Wensi  Zhu,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Petras  Kundrotas,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Victoria  Ruiz Serra,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Carlos H M  Rodrigues,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Alistair S  Dunham,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  David  Burke,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Neera  Borkakoti,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Sameer  Velankar,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Adam  Frost,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Kresten  Lindorff-Larsen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Alfonso  Valencia<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Sergey  Ovchinnikov<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>#</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  David B  Ascher<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Janet M  Thornton<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Norman E  Davey<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Amelie  Stein<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Arne  Elofsson<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Tristan I  Croll<nobr><em>#</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Pedro  Beltrao<nobr><em>#</em></nobr>
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Nature Structural &amp; Molecular Biology
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.nature.com/articles/s41594-022-00849-w" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Most proteins fold into 3D structures that determine how they function and orchestrate the biological processes of the cell. Recent developments in computational methods for protein structure predictions have reached the accuracy of experimentally determined models. Although this has been independently verified, the implementation of these methods across structural-biology applications remains to be tested. Here, we evaluate the use of AlphaFold2 (AF2) predictions in the study of characteristic structural elements; the impact of missense variants; function and ligand binding site predictions; modeling of interactions; and modeling of experimental structural data. For 11 proteomes, an average of 25% additional residues can be confidently modeled when compared with homology modeling, identifying structural features rarely seen in the Protein Data Bank. AF2-based predictions of protein disorder and complexes surpass dedicated tools, and AF2 models can be used across diverse applications equally well compared with experimentally determined structures, when the confidence metrics are critically considered. In summary, we find that these advances are likely to have a transformative impact in structural biology and broader life-science research.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">akdelStructuralBiologyCommunity2022b</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A structural biology community assessment of {AlphaFold2} applications}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{29}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{2022 The Author(s)}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1545-9985}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41594-022-00849-w}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Structural \&amp; Molecular Biology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Akdel&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Mehmet and Pires&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Douglas E V and Porta Pardo&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Eduard and J{\"a}nes&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, J{\"u}rgen and Zalevsky&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Arthur O and M{\'e}sz{\'a}ros&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, B{\'a}lint and Bryant&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Patrick and Good&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Lydia L. and Laskowski&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Roman A and Pozzati, Gabriele and Shenoy, Aditi and Zhu, Wensi and Kundrotas, Petras and Ruiz Serra, Victoria and Rodrigues, Carlos H M and Dunham, Alistair S and Burke, David and Borkakoti, Neera and Velankar, Sameer and Frost, Adam and Lindorff-Larsen, Kresten and Valencia&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Alfonso and Ovchinnikov&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Sergey and Durairaj&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Janani and Ascher&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, David B and Thornton&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Janet M and Davey&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Norman E and Stein&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Amelie and Elofsson&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Arne and Croll&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Tristan I and Beltrao&lt;nobr&gt;&lt;em&gt;#&lt;/em&gt;&lt;/nobr&gt;, Pedro}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Number: 11
  Publisher: Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Protein folding, Research data, Structural biology}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1056--1067}</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">New Phytologist</abbr>
    
  
  </div>

  <div id="wang2022tomato" class="col-sm-8">
    
      <div class="title">The tomato cytochrome P450 CYP712G1 catalyzes the double oxidation of orobanchol en route to the rhizosphere signaling strigolactone, solanacol</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Yanting  Wang,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Hernando G  Suárez Duran,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Robin van Velzen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Kristyna  Flokova,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Che-Yang  Liao,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Aleksandra  Chojnacka,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Stuart  MacFarlane,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  M Eric  Schranz,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Marnix H  Medema,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Aalt DJ van Dijk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Lemeng  Dong,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Harro  Bouwmeester
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In New Phytologist
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1111/nph.18272" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>
    Strigolactones (SLs) are rhizosphere signalling molecules and phytohormones. The biosynthetic pathway of SLs in tomato has been partially elucidated, but the structural diversity in tomato SLs predicts that additional biosynthetic steps are required. Here, root RNA-seq data and co-expression analysis were used for SL biosynthetic gene discovery.
    This strategy resulted in a candidate gene list containing several cytochrome P450s. Heterologous expression in Nicotiana benthamiana and yeast showed that one of these, CYP712G1, can catalyse the double oxidation of orobanchol, resulting in the formation of three didehydro-orobanchol (DDH) isomers.
    Virus-induced gene silencing and heterologous expression in yeast showed that one of these DDH isomers is converted to solanacol, one of the most abundant SLs in tomato root exudate. Protein modelling and substrate docking analysis suggest that hydroxy-orbanchol is the likely intermediate in the conversion from orobanchol to the DDH isomers.
    Phylogenetic analysis demonstrated the occurrence of CYP712G1 homologues in the Eudicots only, which fits with the reports on DDH isomers in that clade. Protein modelling and orobanchol docking of the putative tobacco CYP712G1 homologue suggest that it can convert orobanchol to similar DDH isomers as tomato.
  </p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang2022tomato</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The tomato cytochrome P450 CYP712G1 catalyzes the double oxidation of orobanchol en route to the rhizosphere signaling strigolactone, solanacol}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Yanting and Durairaj, Janani and Su{\'a}rez Duran, Hernando G and van Velzen, Robin and Flokova, Kristyna and Liao, Che-Yang and Chojnacka, Aleksandra and MacFarlane, Stuart and Schranz, M Eric and Medema, Marnix H and van Dijk, Aalt DJ and Dong, Lemeng and Bouwmeester, Harro}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{New Phytologist}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Wiley Online Library}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
</ol>
<h3 class="bibliography">2021</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">PLOS Comp. Bio.</abbr>
    
  
  </div>

  <div id="durairaj2021integrating" class="col-sm-8">
    
      <div class="title">Integrating structure-based machine learning and co-evolution to investigate specificity in plant sesquiterpene synthases</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Elena  Melillo,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Harro J  Bouwmeester,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jules  Beekwilder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dick de Ridder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Aalt DJ van Dijk
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In PLOS Computational Biology
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1371/journal.pcbi.1008197" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://git.wur.nl/durai001/sts_cation_prediction" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Sesquiterpene synthases (STSs) catalyze the formation of a large class of plant volatiles called sesquiterpenes. While thousands of putative STS sequences from diverse plant species are available, only a small number of them have been functionally characterized. Sequence identity-based screening for desired enzymes, often used in biotechnological applications, is difficult to apply here as STS sequence similarity is strongly affected by species. This calls for more sophisticated computational methods for functionality prediction. We investigate the specificity of precursor cation formation in these elusive enzymes. By inspecting multi-product STSs, we demonstrate that STSs have a strong selectivity towards one precursor cation. We use a machine learning approach combining sequence and structure information to accurately predict precursor cation specificity for STSs across all plant species. We combine this with a co-evolutionary analysis on the wealth of uncharacterized putative STS sequences, to pinpoint residues and distant functional contacts influencing cation formation and reaction pathway selection. These structural factors can be used to predict and engineer enzymes with specific functions, as we demonstrate by predicting and characterizing two novel STSs from Citrus bergamia.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">durairaj2021integrating</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Integrating structure-based machine learning and co-evolution to investigate specificity in plant sesquiterpene synthases}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj, Janani and Melillo, Elena and Bouwmeester, Harro J and Beekwilder, Jules and de Ridder, Dick and van Dijk, Aalt DJ}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Computational Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e1008197}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science San Francisco, CA USA}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li></ol>
<h3 class="bibliography">2020</h3>
<ol class="bibliography">
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">CSBJ</abbr>
    
  
  </div>

  <div id="akdel2020caretta" class="col-sm-8">
    
      <div class="title">Caretta–A multiple protein structure alignment and feature extraction suite</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dick de Ridder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Aalt DJ van Dijk
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Computational and Structural Biotechnology Journal
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1016/j.csbj.2020.03.011" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://git.wur.nl/durai001/caretta" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.bioinformatics.nl/caretta" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The vast number of protein structures currently available opens exciting opportunities for machine learning on proteins, aimed at predicting and understanding functional properties. In particular, in combination with homology modelling, it is now possible to not only use sequence features as input for machine learning, but also structure features. However, in order to do so, robust multiple structure alignments are imperative. Here we present Caretta, a multiple structure alignment suite meant for homologous but sequentially divergent protein families which consistently returns accurate alignments with a higher coverage than current state-of-the-art tools. Caretta is available as a GUI and command-line application and additionally outputs an aligned structure feature matrix for a given set of input structures, which can readily be used in downstream steps for supervised or unsupervised machine learning. We show Caretta’s performance on two benchmark datasets, and present an example application of Caretta in predicting the conformational state of cyclin-dependent kinases.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">akdel2020caretta</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Caretta--A multiple protein structure alignment and feature extraction suite}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Akdel&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Mehmet and Durairaj&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Janani and de Ridder, Dick and van Dijk, Aalt DJ}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computational and Structural Biotechnology Journal}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{18}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{981--992}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Bioinformatics</abbr>
    
  
  </div>

  <div id="durairaj2020geometricus" class="col-sm-8">
    
      <div class="title">Geometricus represents protein structures as shape-mers derived from moment invariants</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dick de Ridder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Aalt DJ van Dijk
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Bioinformatics
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1093/bioinformatics/btaa839" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/TurtleTools/geometricus" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>As the number of experimentally solved protein structures rises, it becomes increasingly appealing to use structural information for predictive tasks involving proteins. Due to the large variation in protein sizes, folds and topologies, an attractive approach is to embed protein structures into fixed-length vectors, which can be used in machine learning algorithms aimed at predicting and understanding functional and physical properties. Many existing embedding approaches are alignment based, which is both time-consuming and ineffective for distantly related proteins. On the other hand, library- or model-based approaches depend on a small library of fragments or require the use of a trained model, both of which may not generalize well. We present Geometricus, a novel and universally applicable approach to embedding proteins in a fixed-dimensional space. The approach is fast, accurate, and interpretable. Geometricus uses a set of 3D moment invariants to discretize fragments of protein structures into shape-mers, which are then counted to describe the full structure as a vector of counts. We demonstrate the applicability of this approach in various tasks, ranging from fast structure similarity search, unsupervised clustering and structure classification across proteins from different superfamilies as well as within the same family.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">durairaj2020geometricus</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Geometricus represents protein structures as shape-mers derived from moment invariants}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj, Janani and Akdel, Mehmet and de Ridder, Dick and van Dijk, Aalt DJ}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Bioinformatics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{36}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{Supplement\_2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{i718--i725}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Arch. <br>Biochem. <br>Biophys.</abbr>
    
  
  </div>

  <div id="di2020santalene" class="col-sm-8">
    
      <div class="title">The santalene synthase from Cinnamomum camphora: Reconstruction of a sesquiterpene synthase from a monoterpene synthase</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Alice  Di Girolamo<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Adèle van Houwelingen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Francel  Verstappen,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dirk  Bosch,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Katarina  Cankar,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Harro  Bouwmeester,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dick de Ridder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Aalt DJ van Dijk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Jules  Beekwilder
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Archives of Biochemistry and Biophysics
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1016/j.abb.2020.108647" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Plant terpene synthases (TPSs) can mediate formation of a large variety of terpenes, and their diversification contributes to the specific chemical profiles of different plant species and chemotypes. Plant genomes often encode a number of related terpene synthases, which can produce very different terpenes. The relationship between TPS sequence and resulting terpene product is not completely understood. In this work we describe two TPSs from the Camphor tree Cinnamomum camphora (L.) Presl. One of these, CiCaMS, acts as a monoterpene synthase (monoTPS), and mediates the production of myrcene, while the other, CiCaSSy, acts as a sesquiterpene synthase (sesquiTPS), and catalyses the production of α-santalene, β-santalene and trans-α-bergamotene. Interestingly, these enzymes share 97% DNA sequence identity and differ only in 22 amino acid residues out of 553. To understand which residues are essential for the catalysis of monoterpenes resp. sesquiterpenes, a number of hybrid synthases were prepared, and supplemented by a set of single-residue variants. These were tested for their ability to produce monoterpenes and sesquiterpenes by in vivo production of sesquiterpenes in E. coli, and by in vitro enzyme assays. This analysis pinpointed three residues in the sequence which could mediate the change in product specificity from a monoterpene synthase to a sesquiterpene synthase. Another set of three residues defined the sesquiterpene product profile, including the ratios between sesquiterpene products.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">di2020santalene</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The santalene synthase from Cinnamomum camphora: Reconstruction of a sesquiterpene synthase from a monoterpene synthase}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Di Girolamo&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Alice and Durairaj&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Janani and van Houwelingen, Ad{\`e}le and Verstappen, Francel and Bosch, Dirk and Cankar, Katarina and Bouwmeester, Harro and de Ridder, Dick and van Dijk, Aalt DJ and Beekwilder, Jules}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Archives of Biochemistry and Biophysics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{695}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{108647}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Academic Press}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
</ol>
<h3 class="bibliography">2019</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Phytochemistry</abbr>
    
  
  </div>

  <div id="durairaj2019analysis" class="col-sm-8">
    
      <div class="title">An analysis of characterized plant sesquiterpene synthases</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Alice  Di Girolamo<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Harro J  Bouwmeester,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dick de Ridder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jules  Beekwilder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Aalt DJ van Dijk
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Phytochemistry
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1016/j.phytochem.2018.10.020" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.biorxiv.org/content/10.1101/466839v1" role="button" target="_blank" rel="noopener noreferrer">Preprint</a>
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.bioinformatics.nl/sesquiterpene/synthasedb/" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Plants exhibit a vast array of sesquiterpenes, C15 hydrocarbons which often function as herbivore-repellents or pollinator-attractants. These in turn are produced by a diverse range of sesquiterpene synthases. A comprehensive analysis of these enzymes in terms of product specificity has been hampered by the lack of a centralized resource of sufficient functionally annotated sequence data. To address this, we have gathered 262 plant sesquiterpene synthase sequences with experimentally characterized products. The annotated enzyme sequences allowed for an analysis of terpene synthase motifs, leading to the extension of one motif and recognition of a variant of another. In addition, putative terpene synthase sequences were obtained from various resources and compared with the annotated sesquiterpene synthases. This analysis indicated regions of terpene synthase sequence space which so far are unexplored experimentally. Finally, we present a case describing mutational studies on residues altering product specificity, for which we analyzed conservation in our database. This demonstrates an application of our database in choosing likely-functional residues for mutagenesis studies aimed at understanding or changing sesquiterpene synthase product specificity.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">durairaj2019analysis</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{An analysis of characterized plant sesquiterpene synthases}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Janani and Di Girolamo&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Alice and Bouwmeester, Harro J and de Ridder, Dick and Beekwilder, Jules and van Dijk, Aalt DJ}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phytochemistry}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{158}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{157--165}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li></ol>
<h2 class="bibliography">conference articles &amp; others</h2>
<h3 class="bibliography">2024</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">ICML</abbr>
    
  
  </div>

  <div id="durairaj24plinder" class="col-sm-8">
    
      <div class="title">PLINDER: The Protein-Ligand Interactions Dataset and Evaluation Resource</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Yusuf  Adeshina<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Zhonglin  Cao,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Xuejin  Zhang,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Vladas  Oleinikovas,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Thomas  Duignan,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Zachary  McClure,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Xavier  Robin,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Daniel  Kovtun,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Emanuele  Rossi,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Guoqing  Zhou,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Srimukh  Veccham,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Clemens  Isert,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Yuxing  Peng,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Prabindh  Sundareson,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriele  Corso,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Hannes  Stärk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerardo  Tauriello,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Zachary  Carpenter,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Michael  Bronstein,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Emine  Kucukbenli,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Luca  Naef
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In ICML’24 Workshop ML for Life and Material Science: From Theory to Industry Applications
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1101/2024.07.17.603955" role="button" target="_blank" rel="noopener noreferrer">Preprint</a>
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/plinder-org/plinder" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://plinder.sh" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Protein-ligand interactions (PLI) are foundational to small molecule drug design. With computational methods striving towards experimental accuracy, there is a critical demand for a well-curated and diverse PLI dataset. Existing datasets are often limited in size and diversity, and commonly used evaluation sets suffer from training information leakage, hindering the realistic assessment of method generalization capabilities. To address these shortcomings, we present PLINDER, the largest and most annotated dataset to date, comprising 449,383 PLI systems, each with over 500 annotations, similarity metrics at protein, pocket, interaction and ligand levels, and paired unbound (apo) and predicted structures. We propose an approach to generate training and evaluation splits that minimizes task-specific leakage and maximizes test set quality, and compare the resulting performance of DiffDock when retrained with different kinds of splits.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
  </div>
</div></li></ol>
<h3 class="bibliography">2023</h3>
<ol class="bibliography">
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Springer Book</abbr>
    
  
  </div>

  <div id="durairajGenomesVariantInterpretations2023" class="col-sm-8">
    
      <div class="title">From Genomes to Variant Interpretations Through Protein Structures</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Leila Tamara  Alexander<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gabriel  Studer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerardo  Tauriello,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Ingrid Guarnetti  Prandi,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Rosalba  Lepore,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Giovanni  Chillemi,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Torsten  Schwede
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            2023
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://doi.org/10.1007/978-3-031-30691-4_6" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The large amount of genetic, phenotypic, and structural data from diverse conditions and environments offers opportunities for new groundbreaking research. Today, the major scientific task is to interpret the vast number of genetic variants within these data. As described in this chapter, identifying relevant variants and connecting them with the associated protein structural and environmental information is a powerful approach to biological discoveries. The unified view of the data brings us a step closer to understanding genetic variation, which is also fundamental for achieving the goals of personalized medicine and the planet’s environment.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">durairajGenomesVariantInterpretations2023</span><span class="p">,</span>
  <span class="na">address</span> <span class="p">=</span> <span class="s">{Cham}</span><span class="p">,</span>
  <span class="na">series</span> <span class="p">=</span> <span class="s">{{SpringerBriefs} in {Applied} {Sciences} and {Technology}}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{From {Genomes} to {Variant} {Interpretations} {Through} {Protein} {Structures}}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{978-3-031-30691-4}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{{Exscalate4CoV}: {High}-{Performance} {Computing} for {COVID} {Drug} {Discovery}}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer International Publishing}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Janani and Alexander&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Leila Tamara and Studer, Gabriel and Tauriello, Gerardo and Prandi, Ingrid Guarnetti and Lepore, Rosalba and Chillemi, Giovanni and Schwede, Torsten}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Coletti, Silvano and Bernardi, Gabriella}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-3-031-30691-4_6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{41--50}</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">ACM <br>Computing Frontiers</abbr>
    
  
  </div>

  <div id="palermoTunablePortableExtremeScale2023" class="col-sm-8">
    
      <div class="title">Tunable and Portable Extreme-Scale Drug Discovery Platform at Exascale: the LIGATE Approach</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Gianluca  Palermo,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gianmarco  Accordi,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Davide  Gadioli,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Emanuele  Vitali,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Cristina  Silvano,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Bruno  Guindani,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Danilo  Ardagna,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andrea R.  Beccari,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Domenico  Bonanni,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Carmine  Talarico,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Filippo  Lunghini,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jan  Martinovic,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Paulo  Silva,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Ada  Bohm,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jakub  Beranek,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Jan  Krenek,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Branislav  Jansik,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Luigi  Crisci,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                    Biagio,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                    Cosenza,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Peter  Thoman,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Philip  Salzmann,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Thomas  Fahringer,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Leila  Alexander,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Gerardo  Tauriello,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Torsten  Schwede,
                
              
            
          
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj</em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Andrew  Emerson,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Federico  Ficarelli,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Sebastian  Wingbermuhle,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Eric  Lindahl,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Daniele  Gregori,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Emanuele  Sana,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Silvano  Coletti,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Philip  Gschwandtner
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In 20th ACM International Conference on Computing Frontiers (CF’23)
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://arxiv.org/abs/2304.09953v1" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Today digital revolution is having a dramatic impact on the pharmaceutical industry and the entire healthcare system. The implementation of machine learning, extreme-scale computer simulations, and big data analytics in the drug design and development process offers an excellent opportunity to lower the risk of investment and reduce the time to the patient. Within the LIGATE project, we aim to integrate, extend, and co-design best-in-class European components to design Computer-Aided Drug Design (CADD) solutions exploiting today’s high-end supercomputers and tomorrow’s Exascale resources, fostering European competitiveness in the field. The proposed LIGATE solution is a fully integrated workflow that enables to deliver the result of a virtual screening campaign for drug discovery with the highest speed along with the highest accuracy. The full automation of the solution and the possibility to run it on multiple supercomputing centers at once permit to run an extreme scale in silico drug discovery campaign in few days to respond promptly for example to a worldwide pandemic crisis.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">palermoTunablePortableExtremeScale2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Tunable and {Portable} {Extreme}-{Scale} {Drug} {Discovery} {Platform} at {Exascale}: the {LIGATE} {Approach}}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Tunable and {Portable} {Extreme}-{Scale} {Drug} {Discovery} {Platform} at {Exascale}}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-10-10}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv.org}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Palermo, Gianluca and Accordi, Gianmarco and Gadioli, Davide and Vitali, Emanuele and Silvano, Cristina and Guindani, Bruno and Ardagna, Danilo and Beccari, Andrea R. and Bonanni, Domenico and Talarico, Carmine and Lunghini, Filippo and Martinovic, Jan and Silva, Paulo and Bohm, Ada and Beranek, Jakub and Krenek, Jan and Jansik, Branislav and Crisci, Luigi and Biagio and Cosenza and Thoman, Peter and Salzmann, Philip and Fahringer, Thomas and Alexander, Leila and Tauriello, Gerardo and Schwede, Torsten and Durairaj, Janani and Emerson, Andrew and Ficarelli, Federico and Wingbermuhle, Sebastian and Lindahl, Eric and Gregori, Daniele and Sana, Emanuele and Coletti, Silvano and Gschwandtner, Philip}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">conference</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">eventtitle</span> <span class="p">=</span> <span class="s">{20th ACM International Conference on Computing Frontiers (CF'23)}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1145/3587135.3592172}</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li>
</ol>
<h3 class="bibliography">2021</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">MLSB NeurIPS</abbr>
    
  
  </div>

  <div id="durairaj2021fast" class="col-sm-8">
    
      <div class="title">Fast and adaptive protein structure representations for machine learning</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                <em class="author">Janani  Durairaj<nobr><em>*</em></nobr></em>,
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Mehmet  Akdel<nobr><em>*</em></nobr>,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Dick de Ridder,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  and Aalt DJ van Dijk
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            In Machine Learning for Structural Biology Workshop, NeurIPS 2020
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
          <a class="abstract badge bib-button waves-effect font-weight-light mr-1" role="button">Abstract</a>
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.biorxiv.org/content/10.1101/2021.04.07.438777v1" role="button" target="_blank" rel="noopener noreferrer">Preprint</a>
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://github.com/TurtleTools/caretta" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://www.bioinformatics.nl/caretta" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The growing prevalence and popularity of protein structure data, both experimental and computationally modelled, necessitates fast tools and algorithms to enable exploratory and interpretable structure-based machine learning. Alignment-free approaches have been developed for divergent proteins, but proteins sharing func-tional and structural similarity are often better understood via structural alignment, which has typically been too computationally expensive for larger datasets. Here, we introduce the concept of rotation-invariant shape-mers to multiple structure alignment, creating a structure aligner that scales well with the number of proteins and allows for aligning over a thousand structures in 20 minutes. We demonstrate how alignment-free shape-mer counts and aligned structural features, when used in machine learning tasks, can adapt to different levels of functional hierarchy in protein kinases, pinpointing residues and structural fragments that play a role in catalytic activity.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">durairaj2021fast</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Fast and adaptive protein structure representations for machine learning}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Janani and Akdel&lt;nobr&gt;&lt;em&gt;*&lt;/em&gt;&lt;/nobr&gt;, Mehmet and de Ridder, Dick and van Dijk, Aalt DJ}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">eventtitle</span> <span class="p">=</span> <span class="s">{Machine Learning for Structural Biology Workshop, NeurIPS 2020}</span><span class="p">,</span>
  <span class="na">conference</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li></ol>
<h3 class="bibliography">2020</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Course</abbr>
    
  
  </div>

  <div id="kootstra2020crash" class="col-sm-8">
    
      <div class="title">Crash Course Machine Learning</div>
      <div class="author">
        
          
          
          
          

          
          
            
              
                
                  Geert W  Kootstra,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  Aalt DJ van Dijk,
                
              
            
          
        
          
          
          
          

          
          
            
              
                
                  David Rapado  Rincon,
                
              
            
          
        
          
          
          
          

          
          
            
              
                and <em class="author">Janani  Durairaj</em>
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            2020
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://research.wur.nl/en/publications/crash-course-machine-learning" target="_blank" rel="noopener noreferrer">HTML</a>
        
        
        
        
        
        
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://git.wur.nl/durai001/data_science_crash_course" target="_blank" rel="noopener noreferrer">Code</a>
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
      </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">kootstra2020crash</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Crash Course Machine Learning}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kootstra, Geert W and van Dijk, Aalt DJ and Rincon, David Rapado and Durairaj, Janani}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li></ol>
<h2 class="bibliography">theses</h2>
<h3 class="bibliography">2021</h3>
<ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">PhD Thesis</abbr>
    
  
  </div>

  <div id="durairaj2021computational" class="col-sm-8">
    
      <div class="title">Computational approaches to discover novel enzymes for fragrance and flavour</div>
      <div class="author">
        
          
          
          
          

          
          
            
              <em class="author">Janani  Durairaj</em>
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
        </p>
      
      
      </div>
    

    <div class="col p-0">
        
        
        
        
        
        
        
        
        
        
        
        
          <a class="bibtex badge bib-button waves-effect font-weight-light mr-1" role="button">BibTeX</a>
        
        
          <a class="badge bib-button waves-effect font-weight-light mr-1" href="https://research.wur.nl/en/publications/computational-approaches-to-discover-novel-enzymes-for-fragrance-" role="button" target="_blank" rel="noopener noreferrer">Website</a>
        
      </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    
    <div class="bibtex hidden">
        
        <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@phdthesis</span><span class="p">{</span><span class="nl">durairaj2021computational</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Computational approaches to discover novel enzymes for fragrance and flavour}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Durairaj, Janani}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">school</span> <span class="p">=</span> <span class="s">{Wageningen University}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
    </div>
    
  </div>
</div></li></ol>

</div>

  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    © Copyright 2025 Janani  Durairaj.
    Powered by <a href="http://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>.

    
    
    Last updated: February 14, 2025.
    
  </div>
</footer>



  </body>

  <!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Medium Zoom JS -->
<script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
<script src="/assets/js/zoom.js"></script>


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>

  
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  





</html>
